persistence/src/main/java/com/epam/esm/persistence/util/SortDirection.java

```diff
@@ -7,5 +7,15 @@
  * @version 1.0
  */
 public enum SortDirection {
-    ASC, DESC
+    ASC, DESC;
+
+    public static SortDirection parseAscDesc(String param) {
+        switch (param) {
+            case "2":
```

Comment:
give name for the constant "2"

Reply:
private final static String DESC_DIGIT = "2";